### PR TITLE
REF - Split upgrade SQL into multiple tasks

### PR DESCRIPTION
**Pull request by colemanw**
Overview
----------------------------------------
Follow up to #22925. Splits up potentially long-running upgrade SQL which alters table columns to be NOT NULL.

Before
----------------------------------------
Lots of SQL in the upgrade file, all runs at once.

After
----------------------------------------
Columns to upgrade is in a more concise array. Runs each as a separate task to avoid timeouts.

Comments
-----------
Ping @monishdeb

### CODE REVIEW

File: CRM/Upgrade/Incremental/php/FiveFortyNine.php [hunk 1]
```diff
@@ -29,6 +62,23 @@ class CRM_Upgrade_Incremental_php_FiveFortyNine extends CRM_Upgrade_Incremental_
...
+  /**
+   * Converts a boolean table column to be NOT NULL
+   * @param CRM_Queue_TaskContext $ctx
+   * @param string $tableName
+   * @param string $columnName
+   * @param string $defn
+   */
+  public static function changeBooleanColumn(CRM_Queue_TaskContext $ctx, $tableName, $columnName, $defn) {
+    CRM_Core_DAO::executeQuery("UPDATE `$tableName` SET `$columnName` = 0 WHERE `$columnName` IS NULL");
+    CRM_Core_DAO::executeQuery("ALTER TABLE `$tableName` CHANGE `$columnName` `$columnName` tinyint NOT NULL $defn");
```
Comment (monishdeb) [inline CRM/Upgrade/Incremental/php/FiveFortyNine.php:81]: need to pass extra arguments for executeQuery
```php
CRM_Core_DAO::executeQuery("ALTER TABLE `$tableName` CHANGE `$columnName` `$columnName` tinyint NOT NULL $defn", TRUE, NULL, FALSE, FALSE);
```

**Comment by colemanw** [comment]
@monishdeb thanks. I've updated the params to skip localization. I don't think there should be any localized columns (because how would you localize a `1` or a `0`??) and hopefully that same unit test will catch any errors if there are.

**Comment by colemanw** [comment]
FYI @monishdeb the regex find-replace in PHPStorm is really nice. I was able to convert the sql file into a php array with this:
- **Find:** ```CHANGE `[^`]+` `(\w+)` tinyint NOT NULL (.*),```
- **Replace:** `'$1' => "$2",`

**Comment by monishdeb** [comment]
Tested, looks good.

**Comment by monishdeb** [comment]
Thanks for the regex and replacement pattern. It really reduced my time to replace the SQLs to array

before

**Pull request merged**